themes/rad/layouts/index.html

```diff
@@ -4,25 +4,25 @@
     <div id="peer">
       <h2>Peer–to–peer</h2>
       <p>
-        The Radicle stack builds on IPFS, which means it works without a centralized server. No hassle with setting up
-        your own hosting, and no need to trust companies with your data.
+        Open source development is a peer&ndash;to&ndash;peer process.
+        With Radicle you can collaborate directly, without giving your
+        data to third parties or setting up your own server.
```

Comment:
Here I prefer the one that mentions explicitly IPFS. I don´t get the "open source development is a peer to peer process¨, Feels too vague to me.